nan_json.h

```diff
@@ -66,18 +67,20 @@ class JSON {
   static v8::Local<v8::Value> Call(const char *method,
     int argc, v8::Local<v8::Value> *argv) {
     v8::Local<v8::Value> globalJSON =
-      Nan::GetCurrentContext()->Global()->Get(
+      Nan::Get(
+        Nan::GetCurrentContext()->Global(),
         Nan::New("JSON").ToLocalChecked()
-      );
+      ).ToLocalChecked();
 
     if (!globalJSON->IsObject()) {
       return Nan::Undefined();
     }
 
-    v8::Local<v8::Object> json = globalJSON->ToObject();
+    v8::Local<v8::Object> json =
+      Nan::To<v8::Object>(globalJSON).ToLocalChecked();
 
     v8::Local<v8::Value> thisMethod =
-      json->Get(Nan::New(method).ToLocalChecked());
+      Nan::Get(json, Nan::New(method).ToLocalChecked()).ToLocalChecked();
 
     if (thisMethod.IsEmpty() || !thisMethod->IsFunction()) {
```

Comment:
`thisMethod` cannot be empty, since it came from a call to `ToLocalChecked`, so part of this check is unnecessary.
However, you should not be calling `ToLocalChecked` here in the first place, `ToLocalChecked` on an empty `MaybeLocal` produces a hard crash. You should check if the `MaybeLocal` from `Nan::Get` is empty before proceeding.

The `globalJSON`getting above has similar issues.